bears/mp3/MP3CheckBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
```

Comment:
You need to add tests as well. Though I'm not sure how you'll test this linter, this is really special. It will take mp3 as input, which is binary and not just any source file that is plain text.